### ISSUE TITLE
Keep up with the release notes

### ISSUE THREAD

**Issue by dvago** [nathancahill/split#279]
Hi there,

First of all I wanted to thank you for open sourcing the project and maintaining it.

The present is to ask the possibility to keep the releases updated as it seems like the project on GitHub has the last tag on 1.5.9 but the related NPM package was 1.5.11 and has been recently updated to 1.6.1 ref (https://www.npmjs.com/package/split.js). 

No idea what are the changes between these releases

Thanks again for your time.

**Comment by mar10** [comment]
I would like to second this request: it would help if you could make tags for the releases, and maybe even groom a change log.

**Comment by LeeGDavis** [comment]
Yes please, if you can update the tagging to match the releases that would be much appreciated!
